### ISSUE TITLE
console app window gone

### ISSUE THREAD

**Issue by arnaud-carre** [slynch8/10x#2126]
Hi!

When dealing with "console" kind of app in visual studio, when you run the app ( CTRL-F5 ), the console window is waiting for a key at the end of the app. ( so you can look what happen during the execution ).
When running from 10xeditor ( CTRL-F5 ), the console immediatly close at the end.

Is there a way to get same visual studio behavior? ( ie wait for a key )

**Comment by arnaudcarre** [comment]
any update about triaging this issue? Right now you can't use 10xeditor for a simple console application doing like
int main()
{
 printf("hello world\n");
 return 0;
}

because if you run this with CTRL-F5, the application will print and immediatly close, so you won't be able to see "hello world" at all. I'm using tons of small console applications for various tools, and I can't use 10x for these apps :(

**Comment by slynch8** [comment]
Is this for a sln or 10x workspace?

**Comment by slynch8** [comment]
It's not exactly the same as VS, but when you ctrl+F5 in 10x it doesn't launch a cmd window, but launches the exe and logs the output to the build output panel.

**Comment by arnaudcarre** [comment]
it's for .sln. And no, when launching with CTRL-F5 the console app output is *not* displayed in the build output pannel

```
----- Building Project ConsoleApplication1 Release x64 -----
Executing build command:
msbuild /t:"ConsoleApplication1" /p:Configuration="Release" /p:Platform="x64" /verbosity:minimal "D:/UnitySrc/ConsoleApplication1/ConsoleApplication1.sln" /m
MSBuild version 17.5.1+f6fdcf537 for .NET Framework

  ConsoleApplication1.vcxproj -> D:\UnitySrc\ConsoleApplication1\x64\Release\ConsoleApplication1.exe
===== Success : 0 errors, 0 warnings =====
[run] Running: D:\UnitySrc\ConsoleApplication1\x64\Release\ConsoleApplication1.exe 
[run] Exit code: 0

```

(this is the output when running CTRL-F5 of a "printf("hello world\n"); sample

**Comment by slynch8** [comment]
fixed (1.0.81)
